### DISCUSSION TITLE
Issues displaying package list in web UI

### DISCUSSION THREAD

**Discussion by Sov3rain** [verdaccio/verdaccio#3059] [Q&A] [answered]
Hi there!

From updating to Verdaccio 5.6 we have some random issues displaying the package list on the web UI.

I'm connected, but the server keeps throwing at me that i'm not:

![Screenshot 2022-03-11 101125](https://user-images.githubusercontent.com/10755450/157837626-0ed77f2d-4eab-427f-982f-2c44a453adff.jpg)

![photo_2022-03-06_15-49-41](https://user-images.githubusercontent.com/10755450/157837902-37999491-f6c1-48af-95d5-2b56214ddacd.jpg)

My server runs Verdaccio 5.6 on a Google VM Instance, behind a Apache2 proxy, i'm **not** using https.
My authorization plugin is the default one (htpasswd).
My storage path is set to a absolute path, because it doesn't work with the relative one.

When I disconnect then reconnect, everything is back to normal, I can browse the repository.

It's annoying because I have to disconnect then reconnect every 5 to 10 minutes.

Any idea? Thanks!

**Comment by edclement** [comment, marked as answer]
What does the network tab look like in the chrome developer tools? Specifically I would check that the `Authorization` header is present with a JWT token. Also, it would be good to see some matching logs from the Apache proxy. What does the [security](https://verdaccio.org/docs/configuration#security) section of your config file look like?

**Comment by Sov3rain** [comment]
Hey, thank you very much for answering. I took a look at the request headers, and they are present:

![photo_2022-03-12_12-48-12](https://user-images.githubusercontent.com/10755450/158141162-ce95523b-3bd8-44c1-a51f-7a2b24421b50.jpg)

I also took a look in my config.yaml and I have no security section. Is that a new requirement from 5.6?

**Comment by Sov3rain** [comment]
@edclement thanks for pointing me to the security section. I added this block to my config file a few days ago:

```yaml
security:
   web:
     sign:
       expiresIn: 29d
```

So far so good!

**Reply by juanpicado** [reply]
I wonder if the security section is visible enough, I've encountered many who do not find it so easy.

**Reply by Sov3rain** [reply]
I think the problem is it's not present in the config file default example: https://verdaccio.org/docs/configuration#default-configuration

Many people just copy the first bloc, and call it a day. I don't know for sure, but my config file was auto generated by Verdaccio the first time I ran it, but didn't include this section, maybe you should add it when the config file is auto-generated?

**Reply by juanpicado** [reply]
At this point are not generated, https://github.com/verdaccio/verdaccio/tree/5.x/conf 

We could find a way to add more content to the file for creating awareness, that's an option for sure.

**Reply by Sov3rain** [reply]
Since you do already have defaults for this values, I think exposing them in any way (commented or not) in the generated config file could be a great help!

**Reply by juanpicado** [reply]
I'll add then for the next release, thanks @Sov3rain
